core-scala/src/main/scala/com/baeldung/scala/variance/Variance.scala

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.scala.variance
+
+object Variance {
```

Comment:
Also, since you first added this code, the Scala modules have been renamed and there are some new ones. Seems like this code should go in the scala-core-oop module.